processing/src/main/java/org/apache/druid/frame/file/FrameFile.java

```diff
@@ -63,13 +57,18 @@
  *
  * - 2 bytes: {@link FrameFileWriter#MAGIC}
  * - NNN bytes: sequence of {@link FrameFileWriter#MARKER_FRAME} followed by one compressed frame (see {@link Frame})
+ * - 1 byte: {@link FrameFileWriter#MARKER_NO_MORE_FRAMES}
```

Comment:
Should we also add a frameVersion byte?
We might add more things to this in the future then things like snapshotting for fault tolerance becomes tricky

Reply:
We've got this covered: there's version bytes for each frame, & if we need to version the entire frame file format, we can increment the MAGIC.